woocommerce/payment-gateway/payment-tokens/class-sv-wc-payment-gateway-payment-token.php

```diff
@@ -136,6 +137,32 @@ public function get_id() {
 	}
 
 
+	/**
+	 * Gets the gateway ID for the token.
+	 *
+	 * @since 5.6.0-dev.1
+	 *
+	 * @return string
+	 */
+	public function get_gateway_id() {
```

Comment:
Same here, two separate commits.  The slash in the subject is a good indication of that needing to happen